lib/taskana-core/src/main/java/pro/taskana/configuration/TaskanaEngineConfiguration.java

```diff
@@ -93,8 +93,7 @@
     // List of configured classification types
     protected List<String> classificationTypes = new ArrayList<String>();
 
-    // List of configured classification categories
-    protected List<String> classificationCategories = new ArrayList<String>();
+    protected Map<String, List<String>> classificationCategoriesByType = new HashMap<String, List<String>>();
```

Comment:
please name it classificationCategoriesByTypeMap. Sorry. But this helps me to understand the code more easily. Thank you.